Sources/sourcekit-lsp/main.swift

```diff
@@ -130,6 +130,11 @@ struct Main: ParsableCommand {
   )
   var indexPrefixMappings = [PathPrefixMapping]()
 
+  @Option(
+    name: .customLong("generated-interfaces-path", withSingleDash: true),
```

Comment:
I would prefer this to have two dashes, like all the options below. `index-store-path` and friends only have a single dash to be compatible with `clang` compiler arguments. And if we do that, I think it’s also not necessary to specify a `name` for the `Option`.